assets/src/edit-story/components/checklist/checks/elementLinkTappableRegionTooSmall.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { sprintf, __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Text, THEME_CONSTANTS } from '../../../../design-system';
+import { useStory } from '../../../app';
+import { ChecklistCard } from '../../checklistCard';
+import { filterStoryElements } from '../utils';
+
+const LINK_TAPPABLE_REGION_MIN_WIDTH = 48;
+const LINK_TAPPABLE_REGION_MIN_HEIGHT = 48;
```

Comment:
Once the `new_constants.js` is in the new checklist 😅 we can port these values in here instead so that the source is the same for the copy as it is the check.